backend/sublet/serializers.py

```diff
@@ -61,6 +61,31 @@ class SubletSerializer(serializers.ModelSerializer):
         many=True, queryset=Amenity.objects.all(), required=False
     )
 
+    def validate_publish(self, validated_data, instance=None):
+        fields = [
+            "title",
+            "address",
+            "price",
+            "negotiable",
+            "start_date",
+            "end_date",
+            "expires_at",
+        ]
+
+        newest_fields = [
+            (field, validated_data[field])
+            if field in validated_data
+            else (field, getattr(instance, field))
```

Comment:
`getattr` has a default parameter, so we can do `getattr(instance, field, None)` to simplify a bit!